runtime/doc/lua.txt

```diff
@@ -2574,4 +2585,22 @@ parse({version}, {opts})                                 *vim.version.parse()*
         (table|nil) parsed_version Parsed version table or `nil` if `version`
         is invalid.
 
+range({spec})                                            *vim.version.range()*
+    Parameters: ~
+      • {spec}  (string)
+
+Range:matches({self}, {version})                 *vim.version.Range:matches()*
```

Comment:
Yeah. I'm still not sure if we want `vim.version.range` in any form, because ranges are not part of https://semver.org/ : they are a parallel quasi-spec which I guess is specified by https://devhints.io/semver and/or https://github.com/npm/node-semver#ranges ?